common/environment/environment.cpp

```diff
@@ -2839,10 +2876,13 @@ class CEnvironmentClusterInfo: implements IConstWUClusterInfo, public CInterface
     {
         return ldapPassword.str();
     }
-    bool isQueriesOnly() const
+    virtual bool isQueriesOnly() const override
     {
```

Comment:
Did you look at the places isQueriesOnly() is called to make sure nothing would break because we don't always return false?  I would consider removing isQueriesOnly() and changing the calls to getRoxieTargetType() then it would be clear that the logic was correct.

I quickly glanced at the calls I found and one should be weeding out RTTQueries but not RTTBoth and the others should be checking for RTTQueries OR RTTBoth.

Reply:
The isQueriesOnly() is removed.